### ISSUE TITLE
[Package Request] - php-xdebug

### ISSUE THREAD

**Issue by metametadata** [amazonlinux/amazon-linux-2023#651]
**What package is missing from Amazon Linux 2023? Please describe and include package name.**

`php-xdebug` (https://xdebug.org/docs/install#linux)

**Is this an update to existing package or new package request?**

New package request.

**Is this package available in Amazon Linux 2? If it is available via external sources such as EPEL, please specify.**

🤷 

**Any additional information you'd like to include. (use-cases, etc)**

This is how we currently install it: `pecl install xdebug`.

**Comment by adelkedjour** [comment]
You need to install it using `pecl` after installing the pear library `dnf install -y php-pear` it should work without any issues.

**Comment by metametadata** [comment]
Yes, this is what I do currently. But as far as I understand, it compiles the package from the source which is much slower in comparison to using `dnf`, as requested in the ticket.

**Comment by adelkedjour** [comment]
> Yes, this is what I do currently. But as far as I understand, it compiles the package from the source which is much slower in comparison to using `dnf`, as requested in the ticket.

Many packages are missing tbh not just xdebug. ATM there is no other option.

**Comment by itsFelixH** [comment]
I'd like to add our use case for the [xdebug](https://xdebug.org/) package.

We would like to run multiple PHP applications on AL2023. Our developers are using [PHP Strom](https://www.jetbrains.com/phpstorm/) to develop these applications. xdebug integrates into PHP storm and provides a range of features to improve the PHP development experience.

Once using AL2023 distribution it should ideally provide all essential packages and libraries through its official repositories.


We currently use Amazon Linux 2 (yes, this package available in Amazon Linux 2!) and install xdebug with:
`yum -y install php81-php-pecl-xdebug3` for PHP 8.1
and we are in the process of switching to a newer PHPP version:
`yum -y install php83-php-pecl-xdebug3` for PHP 8.3

We would like the option to install xdebug on AL2023 with `dnf`, e.g.
`dnf install -y php83-php-pecl-xdebug3`

**Comment by henriquemattos** [comment]
Seems like half year passed by and no progress and no active discussion in this topic. Would be great to have XDebug available from `dnf`.

My solution has been to get the source for the last available version, decompress it, and install manually. After following the instructions from https://xdebug.org/docs/install#pecl

```shell
dnf install -y wget && \
    cd /tmp/ && \
    wget https://xdebug.org/files/xdebug-3.4.4.tgz && \
    tar -xzf xdebug-3.4.4.tgz && \
    rm -rf tar -xzf xdebug-3.4.4.tgz && \
    cd xdebug-3.4.4 && phpize && ./configure --enable-xdebug && make && make install
```
